docs/local-development.md

```diff
@@ -549,9 +550,52 @@ When we started the Debugger, we were not sure if we would keep webpack and the
 
 Now that we *have* decided that working in github with webpack has a lot of benefits we could switch to JSX. We are open to switching if someone could help us do it, join the [discussion here](https://github.com/devtools-html/debugger.html/issues/1747).
 
+
+### Ninjas
```

Comment:
Guru. We could get like, a tranquil monk picture.

No partisanship here... :laughing: